src/exports.jl

```diff
@@ -897,6 +897,7 @@ export jacobian_ideal
 export jacobian_matrix
 export jennings_series, has_jennings_series, set_jennings_series
 export johnson_solid
+export jukes_cantor_model
```

Comment:
please collect your exports in `experimental/GraphicalModels/src/GraphicalModels.jl` instead, to keep everything concerning this new experimental project tightly together